src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const allPop = document.querySelectorAll('.population');
+const arrPop = [...allPop].map(number => +number.innerText.split(',').join(''));
```

Comment:
`map()` returns new array, copy is redundant here

Reply:
and try use more obvious convert in case with more operations than 1
bad `+numberString`
good `Number(numberString)`